docs/reference/snapshot-restore/repository-source-only.asciidoc

```diff
@@ -27,6 +27,9 @@ As a result, indices adopting synthetic source cannot be restored. When you rest
  * The mapping of the restored index is empty, but the original mapping is available from the types top
    level `meta` element.
 
+Note that information such as document count, deleted document count and store size are not available for
+source-only indices since these indices do not contain the relevant data structures to retrieve this information from.
```

Comment:
Similarly here re. the phrase `source-only indices`. Also could this just be another bullet point in the list of constraints on indices restored from source-only snapshots?